hermes/src/main/java/in/raveesh/hermes/Util.java

```diff
@@ -0,0 +1,16 @@
+package in.raveesh.hermes;
+
+import android.annotation.SuppressLint;
+import android.util.Log;
+
+/**
+ * Created by sahil on 15/5/15.
+ */
+public class Util {
+    public static final String LOG_TAG = "in.raveesh.hermesexample";
+
+    @SuppressLint("LongLogTag")
+    public static void log(String message) {
```

Comment:
Yup it is basically for easier logging. Have been using loggers like Hugo and Timber so have the habit of cleaner logging. Helps in maintaining log tags etc.

The annotation is to suppress the lint warning of long log tags.

http://grepcode.com/file/repo1.maven.org/maven2/com.android.tools.lint/lint-checks/24.1.0/com/android/tools/lint/checks/LogDetector.java#95